fr.handbrake.ghb.Plugin.IntelMediaSDK.json

```diff
@@ -0,0 +1,119 @@
+                    "VERBOSE=1"
+                ],
+                "make-install-args": [
+                    "VERBOSE=1"
+                ]
+            }
+        },
+        {
+            "name": "libva",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/intel/libva/releases/download/2.7.1/libva-2.7.1.tar.bz2",
+                    "sha256": "a00ff19d9f969259b9784172adad7788dbf3de827d985c5d27c230efd5d98a04"
+                }
+            ],
+            "no-autogen": true,
+            "config-opts": ["--with-drivers-path=/app/extensions/IntelMediaSDK/lib/dri"],
+            "build-options": {
+                "prefix" : "/app/extensions/IntelMediaSDK"
+            }
+        },
+        {
+            "name": "libva-utils",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/intel/libva-utils/releases/download/2.7.1/libva-utils-2.7.1.tar.bz2",
+                    "sha256": "ffcd698a7d4312750d7c9338ba6b26774c7028ee6f9f3cbbf34c122c05bcea9a"
+                }
+            ],
+            "no-autogen": true,
+            "build-options": {
+                "prefix" : "/app/extensions/IntelMediaSDK",
+                "prepend-pkg-config-path": "/app/extensions/IntelMediaSDK/lib/pkgconfig"
+            }
+        },
+        {
+            "name": "intel-media-driver",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/intel/media-driver/archive/intel-media-20.1.1.tar.gz",
+                    "sha256": "dbca37ba8383cd4d1f3e15fa906f03457ebdb5a118d2459d8f9cd477e0f3d4d4"
+                }
+            ],
+            "buildsystem": "cmake",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DMEDIA_RUN_TEST_SUITE=OFF"
+            ],
+            "build-options": {
+                "prefix" : "/app/extensions/IntelMediaSDK",
+                "prepend-pkg-config-path": "/app/extensions/IntelMediaSDK/lib/pkgconfig"
+            }
+        },
+        {
+            "name": "mediasdk",
+            "buildsystem": "cmake",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DBUILD_SAMPLES=OFF",
+                "-DBUILD_TESTS=OFF",
+                "-DBUILD_TOOLS=OFF",
+                "-DBUILD_TESTS=OFF",
+                "-DMFX_ENABLE_SW_FALLBACK=OFF"
+            ],
+            "build-options": {
+                "prefix" : "/app/extensions/IntelMediaSDK",
+                "prepend-pkg-config-path": "/app/extensions/IntelMediaSDK/lib/pkgconfig"
+            },
+            "post-install": [
+                "install -Dp -m 644 ${FLATPAK_BUILDER_BUILDDIR}/fr.handbrake.ghb.Plugin.IntelMediaSDK.appdata.xml --target-directory=${FLATPAK_DEST}/share/appdata",
```

Comment:
You can drop the `${FLATPAK_BUILDER_BUILDDIR}`.

Also: The convention is, to call these files `*.metainfo.xml` (btw: also true for apps now!).

Reply:
The build fails without `${FLATPAK_BUILDER_BUILDDIR}`.  `install` is performed one directory level down from root since `"builddir": true` So without the path, it doesn't find the file.

> Also: The convention is, to call these files *.metainfo.xml

Ah, thanks.  Should I update this now?

Reply:
It's because of the `"builddir": true`, I usually just use `../fr.handbrake.ghb.Plugin.IntelMediaSDK.appdata.xml`.

>  Should I update this now?

Sure. And you could also do it for the app. (Granted, it's minor thing.)

Reply:
> I usually just use ../fr.handbrake.ghb.Plugin.IntelMediaSDK.appdata.xml

I thought of that.  But generally speaking out of tree builds do not have to be in that particular location.  I try not to hard code paths whenever possible.